packages/react-instantsearch-dom/src/components/Link.js

```diff
@@ -17,6 +16,6 @@ export default class Link extends Component {
   };
 
   render() {
-    return <a {...omit(this.props, 'onClick')} onClick={this.onClick} />;
+    return <a {...this.props} onClick={this.onClick} />;
```

Comment:
here I just chose to go for overriding the prop, since there isn't much wrong with having that argument passed twice